src/applications/discharge-wizard/containers/RequestDD214.jsx

```diff
@@ -41,7 +41,8 @@ const RequestDD214 = ({ router }) => {
             <div>
               <h4>
                 Download and fill out{' '}
-                <a href="http://www.esd.whs.mil/Portals/54/Documents/DD/forms/dd/dd0149.pdf">
+                {/* <a href="http://www.esd.whs.mil/Portals/54/Documents/DD/forms/dd/dd0149.pdf"> */}
```

Comment:
Any reason we wouldn't want to just delete this comment? 🙂

Reply:
AHHH nice catch